libs/features/mas/src/merch-quantity-select.js

```diff
@@ -93,10 +95,12 @@ export class MerchQuantitySelect extends LitElement {
             inputValue > 0 &&
             inputValue !== this.selectedValue
         ) {
-            const adjustedInputValue =
+            let adjustedInputValue =
                 this.maxInput && inputValue > this.maxInput
                     ? this.maxInput
                     : inputValue;
+            adjustedInputValue = this.min && adjustedInputValue < this.min
+                ? this.min : adjustedInputValue;
```

Comment:
```suggestion
            let adjustedInputValue = inputValue;
            if (this.maxInput && inputValue > this.maxInput)
                    adjustedInputValue = this.maxInput;
           if (this.min && adjustedInputValue < this.min)
                adjustedInputValue = this.min;
```
I think this makes it a bit more readable, I found the reassignments and ternaries to be a bit hard to read with the compound conditionals ahead of them. Let me know what you think.